WebApiCoreSeed.WebApi/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.WebApi.Middleware
+{
+    /// <summary>
+    /// Middleware to handle global authorization policies
+    /// </summary>
+    public class AuthorizationMiddleware
```

Comment:
@mvazquez-ms I apologize, but I have edited your first comment. Remember this is a public repository.